### PR TITLE
#369: Expand Song and Playlist modal sections by hovering

### PR DESCRIPTION
**Pull request by ravi-kumar-t**
## Description
Added Hover Functionality: Menus on PlaylistCard, PlaylistSidebar, and Playlist (ellipsis button) now open on hover. The "Add to playlist" submenu within ContextMenuPlaylist also responds to hover. This significantly improves user experience by making navigation more intuitive and efficient,

## Checklist

- [x] I have read and followed the [Contributing Guidelines](https://github.com/AntonioMrtz/SpotifyElectron/blob/master/docs/CONTRIBUTING.md).
- [x] I have followed the project's [git convention](https://github.com/AntonioMrtz/SpotifyElectron/blob/master/docs/developer/Git-Convention.md).
- [x] I have tested the changes and verified they work as intended. Automated tests (unit, integration, e2e as applicable) and manual tests (UI interactions, edge cases, error states, etc.).
- [ ] I have set the pull request base to the appropriate `development-vx.xx` branch corresponding to the issue milestone.

## Commit type
`feat`

## Issue

#369

## Proposed Changes

- Implemented hover-to-open functionality for context menus on PlaylistCard, PlaylistSidebar, and the ellipsis button in Playlist.
- Enhanced the "Add to playlist" submenu within ContextMenuPlaylist to open reliably on hover

## Potential Impact
This change will greatly improve the user experience by providing quicker access to menu options without requiring a click

## Screenshots

image

## Assigned

@AntonioMrtz

### CODE REVIEW

**Comment by AntonioMrtz** [comment]
Hi @ravi-kumar-t can you take a look at:
* Frontend tests
* Eslint check
* Prettier format

File: Electron/src/components/AdvancedUIComponents/ContextMenu/Playlist/ContextMenuPlaylist.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 import LoadingCircle from 'components/AdvancedUIComponents/LoadingCircle/LoadingCircle';
 import InfoPopover from 'components/AdvancedUIComponents/InfoPopOver/InfoPopover';
 import { InfoPopoverType } from 'components/AdvancedUIComponents/InfoPopOver/types/InfoPopover';
 import { getTokenUsername } from 'utils/token';
 import useFetchGetUserPlaylistNames from 'hooks/useFetchGetUserPlaylistNames';
-import { PlaylistsService } from 'swagger/api';
-import { t } from 'i18next';
-import styles from '../contextMenu.module.css';
-import { PropsContextMenuPlaylist } from '../types/propsContextMenu';
+import { PlaylistsService } from 'swagger/api'; // Ensure this path is correct
```
Comment (AntonioMrtz) [inline Electron/src/components/AdvancedUIComponents/ContextMenu/Playlist/ContextMenuPlaylist.tsx:10]: I don't get this comments

**Review by AntonioMrtz — CHANGES_REQUESTED** [review]
Pipelines

**Comment by AntonioMrtz** [comment]
Inactivity
